From 4a2f77e014b8cdd10af1aac0db40f582be027f51 Mon Sep 17 00:00:00 2001
From: Rabeeh Khoury <rabeeh@solid-run.com>
Date: Sat, 30 Aug 2014 15:17:13 +0300
Subject: [PATCH 1201/1201] Fine tuning CuBox-i and HummingBoard defconfigs

1. Removed evbug as a module from both full blown and base defconfigs
2. Removed # CONFIG_THERMAL_OF is not set since it blocked CONFIG_IMX_THERMAL

Signed-off-by: Rabeeh Khoury <rabeeh@solid-run.com>
---
 arch/arm/configs/imx_v7_cbi_hb_base_defconfig |    2 +-
 arch/arm/configs/imx_v7_cbi_hb_defconfig      |    3 +--
 2 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/arch/arm/configs/imx_v7_cbi_hb_base_defconfig b/arch/arm/configs/imx_v7_cbi_hb_base_defconfig
index 76662f1..3f13468 100644
--- a/arch/arm/configs/imx_v7_cbi_hb_base_defconfig
+++ b/arch/arm/configs/imx_v7_cbi_hb_base_defconfig
@@ -134,7 +134,7 @@ CONFIG_SATA_AHCI_PLATFORM=y
 CONFIG_AHCI_IMX=y
 CONFIG_NETDEVICES=y
 CONFIG_INPUT_EVDEV=y
-CONFIG_INPUT_EVBUG=m
+# CONFIG_INPUT_EVBUG is not set
 CONFIG_KEYBOARD_GPIO=y
 CONFIG_KEYBOARD_IMX=y
 # CONFIG_INPUT_MOUSEDEV_PSAUX is not set
diff --git a/arch/arm/configs/imx_v7_cbi_hb_defconfig b/arch/arm/configs/imx_v7_cbi_hb_defconfig
index 54c7ad5..78d2570 100644
--- a/arch/arm/configs/imx_v7_cbi_hb_defconfig
+++ b/arch/arm/configs/imx_v7_cbi_hb_defconfig
@@ -195,7 +195,7 @@ CONFIG_SATA_AHCI_PLATFORM=y
 CONFIG_AHCI_IMX=y
 CONFIG_NETDEVICES=y
 CONFIG_INPUT_EVDEV=y
-CONFIG_INPUT_EVBUG=m
+# CONFIG_INPUT_EVBUG is not set
 CONFIG_KEYBOARD_GPIO=y
 CONFIG_KEYBOARD_IMX=y
 # CONFIG_INPUT_MOUSEDEV_PSAUX is not set
@@ -4550,7 +4550,6 @@ CONFIG_THERMAL_GOV_FAIR_SHARE=y
 # CONFIG_THERMAL_GOV_USER_SPACE is not set
 CONFIG_THERMAL_GOV_STEP_WISE=y
 # CONFIG_THERMAL_EMULATION is not set
-# CONFIG_THERMAL_OF is not set
 
 CONFIG_INOTIFY=y
 CONFIG_INOTIFY_USER=y
-- 
1.7.9.5

